test/passes/translate-to-fuzz_all-features_metrics_noprint.txt

```diff
@@ -0,0 +1,42 @@
+total
```

Comment:
Hmm, even these numbers are still going to change frequently so diffs will still not be very useful. How about we make this a lit/filecheck test that can just test that all of the expected element types are present without checking their numbers? Even that seems concerning because it looks like there are a large number of expression types missing below.

Reply:
Yeah, I don't think that would quite work - this is random data, so we can't guarantee that all instructions actually appear.

I do think it's still useful to have the metrics. They would show a big unexpected change in a clear way, at least.

Reply:
I think the value of this test is to demonstrate that all of the interesting things we want the fuzzer to generate are in fact generated. I would want the onus to be on whoever changes the fuzzer to find a permutation of the input such that everything is generated at least once. Without that, how can we differentiate random changes in the test output from real bugs?

Reply:
I agree the coverage is not complete now. But finding a random input that generates all the things is only slightly less hard than solving NP-completeness. Well not really, but the method might be "keep adding random stuff until everything is emitted", which may well lead to a very large input.

If you dislike this, I can just remove the `--metrics` part, so it prints nothing. Then it would just test that it doesn't crash. And we'd need to be careful when landing any change to the fuzzer (basically, we'd want to do some manual testing at those times). I think the tradeoff is better the way it is now, but I don't feel strongly, and I see that this is a test that doesn't cleanly cover all the things we'd want, which is I guess what annoys you, and I can sort of see that too.

Reply:
No, I think this is a big improvement; thank you :) Let's keep this as-is for now.